docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -162,13 +163,13 @@ Optional parameters:
 
 **Usage notes:**
 
-* ``join()`` does not consume nodes by design.
+* ``join()`` does not consume nodes, so the results of a ``join()`` operation will include both the original working set as well as the resulting set of nodes.
```

Comment:
:+1: I was going to add a comment about this. The most confusing part (to me) about `limit` is that some of the operators call `take` and others do not.

Reply:
@vertexmc - in discussions with @therealsilence - this came down to two distinct use cases. The first case is a operator which performs some sort of lift which adds nodes to the current set of nodes (lift style).  The second case is a operator which consumes all of the current working set and then does stuff with those nodes (pivot style).

Some of the confusion is definitely around pivot style operations which do NOT consume nodes; so we decided that the limit **parameter** for lift style operations would limit the number of nodes which are added to the current working set. For pivot style operations the limit **parameter** would be a limit on the number of nodes pivoted to.  The ``limit()`` operator itself would continue to be a hard limit on the number of nodes it returns.  That seemed to be the most consistent way to implement a limit parameter across storm.